src/Keboola/Console/Command/DeleteGoodDataProjects.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Keboola\Console\Command;
+
+use Console_CommandLine_Exception;
+use Exception;
+use Keboola\Csv\CsvFile;
+use Keboola\ManageApi\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DeleteGoodDataProjects extends Command
+{
+
+    const ARG_SOURCE_FILE = 'file';
+    const OPTION_FORCE = 'force';
+
+    protected function configure()
+    {
+        $this
+            ->setName('gooddata:delete-projects')
+            ->setDescription('Delete GoodData project in bulk')
+            ->addArgument(self::ARG_SOURCE_FILE, InputArgument::OPTIONAL, 'source CSV file', 'projects.csv')
+            ->addOption(self::OPTION_FORCE, 'f', InputOption::VALUE_NONE, 'Will actually do the work, otherwise it\'s dry run')
+        ;
+    }
+
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $sourceFile = $input->getArgument(self::ARG_SOURCE_FILE);
+        $output->writeln(sprintf('Fetching projects from "%s"', $sourceFile));
+        $force = $input->getOption(self::OPTION_FORCE);
+        $output->writeln($force ? 'FORCE MODE' : 'DRY RUN');
+
+        if ($force) {
+            sleep(1);
+        }
+
+        $requiredEnvs = [
+            'GOODDATA_LOGIN',
+            'GOODDATA_PASSWORD'
+        ];
+        $optionalEnvs = [
+            'GOODDATA_URL'
+        ];
+        $missing = array_diff_key(array_flip($requiredEnvs), array_filter(getenv()));
+        if (count($missing)) {
+            throw new Exception(sprintf(
+                'Missing env variable: %s',
+                implode(array_keys($missing))
```

Comment:
Možná ještě upravit na `implode(', ', array_keys($missing))`

jinak to vrací např.: `Missing env variable: GOODDATA_LOGINGOODDATA_PASSWORD `

Reply:
jj to je bug! 👍